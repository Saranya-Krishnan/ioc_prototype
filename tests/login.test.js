import React from 'react';
import Login from '../views/src/components/login';
import renderer from 'react-test-renderer';


test('it displays a name', () =>{
    const component = renderer.create(
        <Login user="Testing"/>
    );
    let c = component.toJSON();
    expect(c).toMatchSnapshot();
});