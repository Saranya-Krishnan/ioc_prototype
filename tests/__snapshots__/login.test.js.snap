// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it displays a name 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
  Testing
</button>
`;
